ts/components/conversation/composition/CompositionBox.tsx

```diff
@@ -145,6 +149,7 @@ const getDefaultState = (newConvoId?: string) => {
     ignoredLink: undefined,
     stagedLinkPreview: undefined,
     showCaptionEditor: undefined,
+    htmlDirection: getHTMLDirection(),
```

Comment:
I feel like this should probably be a props given by its parent rather than a state value. It's a bit weird to have a readonly state value

Reply:
Will look into this now

Reply:
Fixed